sdk/identity/identity/src/client/identityClient.ts

```diff
@@ -20,6 +20,10 @@ export class IdentityClient extends ServiceClient {
     super(undefined, options);
 
     this.baseUri = this.authorityHost = options.authorityHost;
+
+    if (!this.baseUri.startsWith("https:")) {
+      throw new Error("The authorityHost address must use the 'https' protocol.");
+    }
```

Comment:
This check might be too restrictive if this client supports ADFS

Reply:
Should I check for it at all, or leave it for now and see if anyone runs into trouble with it?